pkg/apis/pxc/v1/pxc_types.go

```diff
@@ -813,6 +819,10 @@ func (cr *PerconaXtraDBCluster) CheckNSetDefaults(serverVersion *version.ServerV
 			c.ProxySQL.ImagePullPolicy = corev1.PullAlways
 		}
 
+		if cr.CompareVersionWith("1.12.0") >= 0 && len(c.ProxySQL.PXCHandler) == 0 {
+			c.ProxySQL.PXCHandler = "internal"
```

Comment:
Maybe have `"internal"` and `"scheduler"` as consts, and like that, you can briefly describe them, like what "internal" refers to.

Reply:
https://github.com/percona/percona-xtradb-cluster-operator/pull/1436/commits/dc30831103a3c22a52941f0c7484c7b0c9ca3f1d